jekyll/_cci2/quickstart-javascript.md

```diff
@@ -54,42 +54,43 @@ jobs:
   build:
     working_directory: ~/cci-demo-react
     docker:
-      image: node:7.4.0
+      - image: node:7.4.0
     steps:
-      - shell:
-        name: Pre-Dependencies
-        command: mkdir ~/artifacts
+      - run:
+          name: Pre-Dependencies
+          command: mkdir /home/ubuntu/cci-demo-react/artifacts
```

Comment:
There are problems with `~` in local builds for the top-level `working_directory` setting in a job (not in steps) but these were coming from me trying to debug that failure and forgetting to change it back. I'll bring back the tilde here.